masterlist.yaml

```diff
@@ -301,3 +305,6 @@ plugins:
       - <<: *patchAvailable
         subs: [ 'DarkerNights.esp' ]
         condition: 'active("Radiant Clouds and Fogs.esp") and active("DarkerNights.esp") and not active("DarkerNights-Radiant.esp")'
+  - name: 'Scrap Everything.esp'
+    msg: [ *doNotClean ]
```

Comment:
Oh, I see it here: http://forums.nexusmods.com/index.php?/topic/3527530-scrap-everything/?p=33160335 @Sharlikran, what do you make of this? Sound legit?